dashboard/src/app/colors/che-colors.styl

```diff
@@ -55,6 +55,19 @@ $che-loader-active-title-color = $che-navy-color
 $che-loader-background-color = darken($che-medium-blue-color, 75%)
 $che-loader-logs-background-color = darken($stroke-color,65%)
 
+// navbar colors
+$navbar-backgroud-color = #282E44
+$navbar-active-menu-background-color = alpha($white-color, 20%)
+$navbar-border-right-color = #000000
+$navbar-active-menu-mark-color = $che-logo-yellow-color
+$navbar-menu-icon-color = #808B9C
+$navbar-active-menu-icon-color = $light-gray-color
+$navbar-menu-text-color = $mouse-gray-color
+$navbar-menu-dark-text-color = $navbar-menu-icon-color
+$navbar-active-menu-text-color = $light-gray-color
+$navbar-section-title-color = #62687C
```

Comment:
I have added black color into che-colors.json .
Other colors are belong to specific theme of navbar and it doesn't make sense to put them to general colors list. So I move them into separate .styl file.